FND/Places/Addresses.rdf

```diff
@@ -99,29 +120,457 @@
+				<owl:onDataRange rdf:resource="&xsd;string"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>primary address number</rdfs:label>
+		<skos:definition>address component that identifies a building with respect to a given street</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fnd-plc-adr;SecondaryUnit">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-plc-adr;AddressComponent"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-arr-arr;hasConstituent"/>
+				<owl:onClass rdf:resource="&fibo-fnd-plc-adr;SecondaryUnitIndicator"/>
+				<owl:maxQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:maxQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-arr-arr;hasConstituent"/>
+				<owl:onClass rdf:resource="&fibo-fnd-plc-adr;SecondaryUnitDesignator"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>secondary unit</rdfs:label>
+		<skos:definition>address component that identifies an individual secondary unit at a given street address</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fnd-plc-adr;SecondaryUnitDesignator">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-cls;Classifier"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-plc-adr;requiresSecondaryUnitRange"/>
+				<owl:onDataRange rdf:resource="&xsd;boolean"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>secondary unit designator</rdfs:label>
+		<skos:definition>classifier for a smaller structure or component within a larger facility, such as an apartment, office, mail stop, or other similar designation</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>Note that only certain secondary units require a secondary range, such as an apartment number, to complete a delivery point.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fnd-plc-adr;SecondaryUnitIndicator">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-plc-adr;AddressComponent"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&lcc-lr;hasTag"/>
+				<owl:onDataRange rdf:resource="&xsd;string"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>secondary unit indicator</rdfs:label>
+		<skos:definition>index to the specific unit within a secondary unit, such as a building or apartment, at a particular street address</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fnd-plc-adr;StreetAddress">
```

Comment:
I think that would be a separate line, which you could use one of the address lines for ... this is strictly the street address line of an address

Reply:
I would hope it would be possible to represent the house/building name as part of the component-based address based on PUB28.

Reply:
It is not part of the street address according to the standard, it would be on a separate line.  We can add a separate class for it, and a property to allow for its usage, as an alternative to the use of an address line, though.